integration_tests/tests/forseti/controls/inventory.rb

```diff
@@ -0,0 +1,115 @@
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+resources_names = attribute('resources_names')
+
+require 'json'
+control 'inventory' do
+
+    describe "Inventory" do
+        before :context do
+            command("forseti inventory purge 0").result
+            command("forseti inventory create").result
+
+             # This variable cannot be used after all the inventories have been purged..
+            @inventory_id = JSON.parse(command("forseti inventory list").stdout).fetch("id")
+        end
+
+        describe "Create an inventory" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti inventory list").stdout).to match /SUCCESS/
+            end
+
+            it "should be visible in the database" do
+                expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select COUNT(DISTINCT gcp_inventory.inventory_index_id) from gcp_inventory join inventory_index ON inventory_index.id = gcp_inventory.inventory_index_id;\"").stdout).to match /1/
+            end
+
+             it "should be visible in the database" do
+                 expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"SELECT count(DISTINCT resource_data->>'$.lifecycleState') FROM gcp_inventory WHERE category = 'resource' and resource_type = 'project' and resource_data->>'$.lifecycleState' = 'ACTIVE';\"").stdout).to match /1/
+             end
+
+             it "should be visible in the database" do
+                 expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"SELECT count(DISTINCT resource_type) from gcp_inventory where resource_type in ('kms_cryptokey', 'kms_keyring');\"").stdout).to match /#{resources_names.count}/
+             end
+        end
+
+        describe "List an inventory" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti inventory list").stderr).to eq ""
+            end
+
+            it "should be visible in the database" do
+                expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT gcp_inventory.inventory_index_id) from gcp_inventory join inventory_index on inventory_index.id = gcp_inventory.inventory_index_id where inventory_index.id = #{@inventory_id};\"").stdout).to match /1/
+            end
+        end
+
+        describe "Get an inventory" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti inventory get #{@inventory_id}").stderr).to eq ""
```

Comment:
Why check on `stderr` vs checking on `stdout`?  We expect the `get` command to produce an output here, right?  So should check on the output.

Reply:
If there are no errors then the database check confirms that the command ran successfully. Hence, I had removed the checks on `stdout` to avoid redundancy. Added it back.

Reply:
ditto: The check on `stdout` is to verify that the grpc layer is working.  So, please check on the `stdout` here.

Reply:
Why is this still checking on `stderr`?

Reply:
I thought it would be nice to check on both `stdout` and `stderr`. Removed `stderr` checks as requested.